src/overlays/actors/ovl_T_Keitai/ac_t_keitai.c

```diff
@@ -2,42 +2,199 @@
 #include "m_lib.h"
 #include "m_actor_dlftbls.h"
 #include "m_object.h"
+#include "m_rcp.h"
+#include "objects/tol_keitai_1/tol_keitai_1.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
 
+#define THIS ((T_Keitai*)thisx)
+
 void aTKT_actor_ct(Actor* thisx, Game_Play* game_play);
 void aTKT_actor_move(Actor* thisx, Game_Play* game_play);
 void aTKT_actor_draw(Actor* thisx, Game_Play* game_play);
 
-#if 0
+void aTKT_setupAction(T_Keitai* this, s32 processIndex);
+
+typedef struct T_Keitai_Animation_Data {
+    /* 0x0 */ BaseAnimationR* animation;
+    /* 0x4 */ f32 startFrame;
+    /* 0x8 */ f32 endFrame;
+} T_Keitai_Animation_Data; // size = 0xC
```

Comment:
For structs that aren't original names they should be PascalCase
```suggestion
} TKeitaiAnimationData; // size = 0xC
```